api/base.proto

```diff
@@ -152,16 +150,15 @@ message ApiConfigSource {
   ApiType api_type = 1;
   // Multiple cluster names may be provided. If > 1 cluster is defined, clusters
   // will be cycled through if any kind of failure occurs.
-  repeated string cluster_name = 2 [(validate.rules).repeated.min_items = 1];
+  repeated string cluster_name = 2 [ (validate.rules).repeated .min_items = 1 ];
```

Comment:
This seems weird spacing.

Reply:
I think the knob for this is SpacesInContainerLiterals=false, which we can apply to protos
var arr = [ 1, 2, 3 ];         vs.     var arr = [1, 2, 3];

Reply:
I was more worried about the fact that it fragged `(validate.rules).repeated.min_items` to `(validate.rules).repeated .min_items`. Will look into this.